packages/o365/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.16.0"
+  changes:
+    - description: Update the mapping of `message` ECS field in SecurityComplianceAlerts from o365audit.Comments to o365audit.Name.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1
```

```suggestion
      link: https://github.com/elastic/integrations/pull/13918
```

This is happening in more cases. Could you please check this before making PR as ready for review?